src/datasets/utils/py_utils.py

```diff
@@ -194,14 +195,15 @@ def map_nested(
     if not isinstance(data_struct, dict) and not isinstance(data_struct, types):
         return function(data_struct)
 
-    disable_tqdm = bool(logger.getEffectiveLevel() > INFO)
+    disable_tqdm = bool(logger.getEffectiveLevel() > logging.INFO)
```

Comment:
Replacing this line with `disable_tqdm = bool(logging.get_verbosity() == logging.NOTSET)` (this check was introduced in https://github.com/huggingface/datasets/pull/2534) causes 2 tests to fail on Windows, which is very strange.

Reply:
Good job discovering what caused this issue !